runtimes/standalone/src/weights/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod attestation;
+pub mod ctype;
+pub mod delegation;
+pub mod did;
```

Comment:
this is probably a leftover? But we should have separate `WeightInfo` implementations for our runtimes?

Reply:
Yes, this is a leftover. I suppose eventually we would need this for our separate `WeightInfo` implementations as you said. I could append this to this PR?!

Reply:
> Yes, this is a leftover. I suppose eventually we would need this for our separate `WeightInfo` implementations as you said. I could append this to this PR?!

let's do that in a separate PR

Reply:
When we do the "real" benchmarks. We don't want to run the benchmarks on our Workstation

Reply:
Fixed in 42820cd